role/token.go

```diff
@@ -62,18 +62,18 @@ func (r *Token) SetParams(key, value string) error {
 	// r.IP = value
 	case "k":
 		r.KeyID = value
-	// case "p":
-	// r.Principal = value
+	case "p":
+		r.Principal = value
 	case "r":
 		r.Roles = strings.Split(value, ",")
 	case "s":
 		r.Signature = value
-		// case "t":
-		// i, err := strconv.ParseInt(value, 10, 64)
-		// if err != nil {
-		// return err
-		// }
-		// r.TimeStamp = time.Unix(i, 0)
+	case "t":
+		i, err := strconv.ParseInt(value, 10, 64)
+		if err != nil {
+			return errors.Wrap(err, "invalid timestamp")
```

Comment:
Please help to add a test case for this line.

Reply:
Fixed
https://github.com/yahoojapan/athenz-authorizer/pull/71/commits/95fccb9f7d6f4206dd1617486ac2abdbf7d36edc